services/iaas/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.17.0 (2024-12-13)
+
+- **Feature:** Add new methods to manage affinity groups: `CreateAffinityGroup`, `UpdateAffinityGroup`, `DeleteAffinityGroup`, `GetAffinityGroup`, and `ListAffinityGroup`
+- **Feature:** Add new methods to manage backups: `CreateBackup`, `DeleteBackup`, `GetBackup`, `ListBackup`, `RestoreBackup`, `ExecuteBackup`,`UpdateBackup` (_beta_)
+- **Feature:** Add new methods to manage images: `CreateImage`, `DeleteImage`, `GetImage`, `ListImage`,`UpdateImage`
+- **Feature:** Add new methods to manage imageshares: `DeleteImageShare`, `GetImageShare`, `SetImageShare`,`UpdateImageShare`
+- **Feature:** Add new methods to manage imageshare consumers: `DeleteImageShareConsumer`, `GetImageShareConsumer`, `SetImageShare`,`UpdateImageShare`
+- **Feature:** Add new methods to manage project NICs: `GetProjectNIC`, `ListProjectNICs`
+- **Feature:** Add new methods to manage snapshots: `CreateSnapshot` (_beta_), `DeleteSnapshot` (_beta_), `GetSnapshot` (_beta_), `ListSnapshot` (_beta_), `UpdateSnapshot` (_beta_)
+- **Bugfix:** Correctly handle nullable attributes in model types
```

Comment:
Why do some of these have "(_beta_)" after?

Reply:
I've added these, as they new functions were coming from endpoints explicitly with a v1beta prefix. So I interpreted these as beta functionality. If this is not the case, I'll remove the flag again. Or is this generally not mentioned here?